.github/workflows/terraAlphaChartBump.yaml

```diff
@@ -28,20 +31,25 @@ jobs:
       - name: "Find and replace Datarepo chartVersion version in alpha action"
         uses: docker://mikefarah/yq:3
         with:
-          args: yq w -i jade-data-repo/.github/workflows/alpha-promotion.yaml env.chartVersion ${{ steps.chartversion.outputs.version }}
+          args: yq w -i ${MODIFIED_REPO}/${MODIFIED_FILEPATH} env.chartVersion ${{ steps.chartversion.outputs.version }}
 
-      - name: Create pull request
-        uses: broadinstitute/create-pull-request@v3.5.0 # forked from peter-evans/create-pull-request
-        id: create-pr
-        with:
-          token: ${{ secrets.CR_TOKEN }}
-          path: jade-data-repo
-          commit-message: "Datarepo chart version update: ${{ steps.chartversion.outputs.version }}"
-          committer: datarepo-bot <noreply@github.com>
-          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          title: "Datarepo helm chart version update: ${{ steps.chartversion.outputs.version }}"
-          branch: "version-update/${{ steps.chartversion.outputs.version }}"
-          body: |
-            Update versions in **${{ steps.chartversion.outputs.version }}**.
-            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
-          labels: "helm,datarepo,automerge,version-update,skip-ci"
+      - name: "Merge changes to jade-data-repo"
+        env:
+          GITHUB_EMAIL: "broadbot@broadinstitute.org"
+          GITHUB_USERNAME: "broadbot"
+          COMMIT_MESSAGE: "Datarepo chart version update: ${{ steps.chartversion.outputs.version }}"
+        run: |
+          #TODO: pull this into datarepo-actions repo
+          echo "[INFO] git config email to ${GITHUB_EMAIL}"
+          git config --global user.email "${GITHUB_EMAIL}"
+          echo "[INFO] git config name ${GITHUB_USERNAME}"
+          git config --global user.name "${GITHUB_USERNAME}"
+          echo "[INFO] git add file ${MODIFIED_FILEPATH}"
+          git add ${MODIFIED_FILEPATH}
+          echo "[INFO] commit w/ msg: ${COMMIT_MESSAGE}"
+          git commit -m ${COMMIT_MESSAGE}"
+          echo "[INFO] git push ${DESTINATION_BRANCH}"
+          git push origin ${DESTINATION_BRANCH}
+          echo "[INFO] post push"
+          commit=$(git rev-parse HEAD)
+          echo "[INFO] commit: $commit"
```

Comment:
Would this be worth making into an action and calling use so it can be re purposed else where and you just feed it a PAT and username?